### PR TITLE
Change vulnerable component from WhatsApp to android-gif-drawable

### PR DESCRIPTION
**Pull request by zmanion**
CVE-2019-11932 focuses on WhatsApp for Android, however the vulnerability is in the android-gif-drawable library used by (and therefore affecting) many other Android apps. CC @koral--.

### CODE REVIEW

**Comment by zmanion** [comment]
We tried contacting Facebook, no response so far, not certain we've reached their CNA representative.

File: 2019/11xxx/CVE-2019-11932.json [hunk 1]
```diff
@@ -9,20 +9,20 @@
...
                         "product_data": [
                             {
-                                "product_name": "WhatsApp for Android",
+                                "product_name": "android-gif-drawable",
                                 "version": {
                                     "version_data": [
                                         {
                                             "version_affected": "!=>",
-                                            "version_value": "2.19.244"
+                                            "version_value": "1.2.19"
```
Comment (koral--) [inline 2019/11xxx/CVE-2019-11932.json]: CVE-2019-11932 was fixed in 1.2.18
https://github.com/koral--/android-gif-drawable/pull/673

Comment (zmanion) [reply 2019/11xxx/CVE-2019-11932.json]: Fixing. We also referenced the wrong commit.

**Comment by zmanion** [comment]
Hi @koral--, should this also have a CVE ID? Would be happy to talk about how to make CVE a regular part of android-gif-drawable security bug fixes, also GitHub may provide some degree of service for assigning CVE IDs.

File: 2019/19xxx/CVE-2019-19269.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (cve-team) [inline 2019/19xxx/CVE-2019-19269.json]: CNA(s) of user (certcc) do not match owning CNA.

File: 2019/11xxx/CVE-2019-11932.json [hunk 3]
```diff
@@ -9,26 +9,22 @@
         "vendor": {
```
Comment (cve-team) [inline 2019/11xxx/CVE-2019-11932.json]: Invalid JSON file

**Comment by zmanion** [comment]
Cleaner PR in https://github.com/CVEProject/cvelist/pull/2845.

**Pull request closed without merging**